File: week-02/day2/draw-triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

number = input('Give me a number please: ')

for e in range(0, 5):
    print(e * number)

File: week-07/day2/draw-triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

for (var i = 0; i < lineCount; ++i){
    if (i === 0){
        console.log('*');
    } else if (i === 1){
        console.log('**');
    } else if (i === 2){
        console.log('***');
    } else if (i === 3){
        console.log('****');
    };
};